joomla-projects/GSoC17_publishing_workflow
# Changes to frontend to work with reordered states

### Summary of Changes

This is to reflect the changes mage in Pull Request : https://github.com/joomla-projects/GSoC17_publishing_workflow/pull/75

This allows to work single article link work properly with condition values changes in above PR.


### Testing Instructions
Create an article from backend, add menu-item to it. Check the visibility at front-end by changing transitions from backend


### Documentation Changes Required
No
